lib/revision_table_access_check_filter.js

```diff
@@ -81,6 +81,13 @@ module.exports = function(hyper, req, next, options, specInfo) {
                 }
             });
         }
-        return next(hyper, req);
+        return next(hyper, req)
+        .then(function(res) {
+            if (req.query.redirect === false || mwUtil.isNoCacheRequest(req)) {
+                res.status = 200;
+                delete res.headers.location;
```

Comment:
Oh right. I still think it's in our best interest not to change such small details as we have a lot of moving parts. Do you see a compelling reason for removing it (given the usage of `encodeURIComponent` on it) ?